preprocessors/collage-detector/SBRIF.py

```diff
@@ -22,12 +22,28 @@
 class SbRIF:
 
     def __init__(self, t=0.75, c=5, s=10, rescale_size=224):
+        """
+        Initialization of a model instance, t determines the confidence level,
+        c ans s are pixel-level constants, rescale_size determines the size of
```

Comment:
Any other details on c and s?
```suggestion
        c and s are pixel-level constants, rescale_size determines the size of
```